.github/workflows/solo-e2e-test.yml

```diff
@@ -260,4 +260,40 @@ jobs:
           echo "Block is $BLOCK"
           echo "BLOCK=$BLOCK" >> $GITHUB_ENV
 
-      # TODO verify MN using curl and REST API
+      - name: Get firstBlock
+        run: |
+          FIRSTBLOCK=$(grpcurl \
+            -plaintext \
+            -import-path protobuf-sources/src/main/proto \
```

Comment:
Q: Why do both paths need to be imported. The `block-node-protobuf` folder should have everything no?